net/routing/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=       routing
+PLUGIN_VERSION=    0.0.1
```

Comment:
I like 3 number version strings - It is just a question of taste ;)
Lets keep 0.1 for the first public release and 1.x for the first release with breaking changes.